lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1899,25 +1931,41 @@ def _set_prometheus_data(self, event):
         `MetricsEndpointAggregator`, that Prometheus unit is provided
         with the complete set of existing scrape jobs and alert rules.
         """
-        jobs = []  # list of scrape jobs, one per relation
+        jobs = [] + _type_convert_stored(
```

Comment:
I mean, it's really to ensure that the `StoredData` jobs are always part of the "baseline" sent on relation changes (since they may be/are set ad-hoc/dynamically by `nrpe_exporter`), but I'll take it as a clever type hack.